### PR TITLE
Add KyriaCity to the minecraft servers

### PR DESCRIPTION
**Pull request by skrevt**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/kyriacity/manifest.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "server_name": "kyriacity",
+  "nice_name": "KyriaCity",
+  "direct_ip": "kyriacity.net",
+  "supported_languages": [
+     "en", "de"
+  ],
+  "social": {
+    "web": "https://kyriacity.net",
+    "web_shop": "https://shop.kyriacity.net",
```
Comment (JakobBruening) [inline minecraft_servers/kyriacity/manifest.json:10]: Please remove as long as shop is not available.

File: minecraft_servers/kyriacity/manifest.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "server_name": "kyriacity",
+  "nice_name": "KyriaCity",
+  "direct_ip": "kyriacity.net",
+  "supported_languages": [
+     "en", "de"
```
Comment (JakobBruening) [inline minecraft_servers/kyriacity/manifest.json:6]: Formatting, "de" to next line.

**Review by JakobBruening — CHANGES_REQUESTED** [review]
- Please compress all images

File: minecraft_servers/kyriacity/manifest.json (hunk 1 above)
Comment (skrevt) [reply minecraft_servers/kyriacity/manifest.json]: done

File: minecraft_servers/kyriacity/manifest.json (hunk 2 above)
Comment (skrevt) [reply minecraft_servers/kyriacity/manifest.json]: done

**Review by JakobBruening — APPROVED** [review]

**Pull request merged**